docs/DeveloperGuide.md

```diff
@@ -595,27 +599,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | -------- | ------------------------------------------ | ------------------------------ | ----------------------------------------------------------------------- |
 | `* * *`  | user                                       | see usage instructions         | refer to instructions when I forget how to use the App                  |
 | `* * *`  | user                                       | add a new student              |                                                                         |
-| `* * *`  | user                                       | delete a student               | remove students no longer taking the module                             |
-| `* * *`  | user                                       | track all students in my class | so that I will know who is missing from my class                        |
+| `* * *`  | user                                       | delete a student               | remove students no longer taking the module                             |                 |
```

Comment:
Perhaps "remove students who are no longer taking the module"?

Reply:
what do you think I should replace it with though?